dashboard/modules/job/job_manager.py

```diff
@@ -142,8 +142,9 @@ class JobSupervisor:
     Job supervisor actor should fate share with subprocess it created.
     """
 
-    WAIT_FOR_JOB_TERMINATION_S = 3
+    WAIT_FOR_JOB_TERMINATION_S = 5
```

Comment:
what is the behavior if serve.shutdown take more 5s?

Reply:
Addressed! Add an environment variable that increases timeout to 30s.